FlashFeetAPI/src/main/java/org/example/resources/JobRoleController.java

```diff
@@ -20,11 +24,16 @@ public class JobRoleController {
     public Response getAllJobRoles(){
         System.out.println("Test");
         try {
+            DatabaseConnector databaseConnector = new DatabaseConnector();
             return Response.ok(jobRoleService.getAllJobRoles()).build();
         } catch (FailedToGetJobsException e){
             System.err.println(e.getMessage());
 
             return Response.serverError().build();
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        } catch (DatabaseConnectionException e) {
+            throw new RuntimeException(e);
```

Comment:
```suggestion
        } catch (FailedToGetJobsException | SQLException | DatabaseConnectionException e){
            System.err.println(e.getMessage());

            return Response.serverError().build();
        }
```

Reply:
runtime exception is generic, you should return a response so you have control over what gets returned to user